[2024-06-24T10:12:26.810+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T10:12:26.833+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.inject_data_lviv scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-24T10:12:26.844+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.inject_data_lviv scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-24T10:12:26.844+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 4
[2024-06-24T10:12:26.863+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): processing_tasks.inject_data_lviv> on 2024-06-22 00:00:00+00:00
[2024-06-24T10:12:26.872+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=111) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T10:12:26.874+0000] {standard_task_runner.py:63} INFO - Started process 115 to run task
[2024-06-24T10:12:26.874+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'processing_tasks.inject_data_lviv', 'scheduled__2024-06-22T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpagk1dwoc']
[2024-06-24T10:12:26.877+0000] {standard_task_runner.py:91} INFO - Job 49: Subtask processing_tasks.inject_data_lviv
[2024-06-24T10:12:26.934+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.processing_tasks.inject_data_lviv scheduled__2024-06-22T00:00:00+00:00 [running]> on host afc8096e8b86
[2024-06-24T10:12:27.074+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='processing_tasks.inject_data_lviv' AIRFLOW_CTX_EXECUTION_DATE='2024-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-22T00:00:00+00:00'
[2024-06-24T10:12:27.076+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T10:12:27.077+0000] {sql.py:276} INFO - Executing: 
            INSERT INTO measures (timestamp, temp, city) VALUES (to_timestamp(1719223721), 293.9, 'Lviv');
            
[2024-06-24T10:12:27.088+0000] {base.py:84} INFO - Using connection ID 'ps_db_conn' for task execution.
[2024-06-24T10:12:27.100+0000] {base.py:84} INFO - Using connection ID 'ps_db_conn' for task execution.
[2024-06-24T10:12:27.104+0000] {sql.py:487} INFO - Running statement: 
            INSERT INTO measures (timestamp, temp, city) VALUES (to_timestamp(1719223721), 293.9, 'Lviv');
            , parameters: None
[2024-06-24T10:12:27.106+0000] {sql.py:496} INFO - Rows affected: 1
[2024-06-24T10:12:27.109+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T10:12:27.119+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=processing_tasks.inject_data_lviv, run_id=scheduled__2024-06-22T00:00:00+00:00, execution_date=20240622T000000, start_date=20240624T101226, end_date=20240624T101227
[2024-06-24T10:12:27.169+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T10:12:27.191+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24T10:12:27.194+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
